tests/xharness/Microsoft.DotNet.XHarness.iOS.Shared/Execution/ProcessManager.cs

```diff
@@ -259,15 +260,17 @@ static async Task KillTreeAsyncInternal (int pid, ILog log, bool? diagnostics =
 							commands.AppendLine ("detach");
 							commands.AppendLine ("quit");
```

Comment:
is that `quit` still needed ?

Reply:
Maybe not, but I wondered if earlier versions of lldb (on older macOS versions) would still support multiple --source arguments, so I left it there.